src/seql/helpers.clj

```diff
@@ -217,21 +222,56 @@
           (qualify entity field)
           {:spec spec :handler handler}))
 
+(defmethod merge-entity-component :create-mutation
+  [{:keys [table entity] :as schema} {:keys [spec]}]
+  (let [mname (qualify entity spec)
+        handler (fn [params]
+                  (h/insert-into table (h/values [(schema/as-row schema entity params)])))]
+    (update schema :mutations assoc
+            mname
+            {:spec mname :handler handler})))
+
+(defn- keys-spec-fields
+  "Resolve qualified key belonging to an `s/keys` spec."
+  [spec-name]
+  (let [spec-def (s/describe spec-name)]
```

Comment:
You can improve this: if you look at https://github.com/exoscale/specs/blob/master/src/exoscale/specs.cljc#L142 it shows how you can make it work for s/merge s/multi-spec and s/keys. This also has the advantage of being able to resolve deep aliases (s/defs pointing to other specs, `(s/def ::foo ::bar)`).

Reply:
I guess we should move `select-keys` in coax directly which would make this simpler (coax is already pulled here and `specs` is still private for now)

Reply:
fixed!